docs/tutorial.md

```diff
@@ -117,102 +137,111 @@ To quit DuckDB, run the command `Ctrl+D` or type `.exit` in the DuckDB shell.
 ### Operate on Volumes with the CLI
 
 Let's list the volumes.
-```
+
+```sh
 bin/uc volume list --catalog unity --schema default
 ```
 
 You should see a few volumes. Let's get the metadata of one of those volumes.
 
-```
+```sh
 bin/uc volume get --full_name unity.default.json_files
 ```
 
 Now let's list the directories/files in this volume.
 
-```
+```sh
 bin/uc volume read --full_name unity.default.json_files
 ```
+
 You should see two text files listed and one directory. Let's read the content of one of those files.
 
-```
+```sh
 bin/uc volume read --full_name unity.default.json_files --path c.json
 ```
-Voila! You have read the content of a file stored in a volume. We can also list the contents of any subdirectory.
-For e.g.:
 
-```
+Voila! You have read the content of a file stored in a volume. We can also list the contents of any subdirectory. For example:
+
+```sh
 bin/uc volume read --full_name unity.default.json_files --path dir1
 ```
 
 Now let's try creating a new external volume. First physically create a directory with some files in it.
-For example in the project root directory, create a directory `/tmp/myVolume` and put some files in it.
+For example in the project root directory, create a directory `tmp/myVolume` and put some files in it.
```

Comment:
@epps why are you changing `/tmp/myVolume` to `tmp/myVolume`. I believe the intention is to use `/tmp`

Reply:
@tdas, based on the wording _"... in the project root directory ..."_ I'd understood that the `tmp` directory should be in the root of the repo, not the root of the file system. With that said, I _was_ confused because `/tmp` would also make sense. Happy to change back if the correct understanding is the latter (i.e. `/tmp/myVolume`), though I think we should also change the wording to something along the lines of _"... create `myVolume` in `/tmp` ..."_.

Reply:
Yeah, that was the mistake. it should not have said `"... in the project root directory ..."`. please remove that and then keep the `/tmp`